server/routes/termdb.topVariablyExpressedGenes.ts

```diff
@@ -110,13 +110,22 @@ async function computeGenes4nativeDs(
 	 * This is based on the recommendation of this article:
 	 * https://www.frontiersin.org/articles/10.3389/fgene.2021.632620/full.
 	 * This article recommends using interquartile region over variance.*/
+
+	let filter_extreme_values = true
+	if (q.filter_extreme_values == 1) {
+		filter_extreme_values = true
+	} else if (q.filter_extreme_values == 0) {
+		filter_extreme_values = false
+	}
+
 	const input_json = {
 		input_file: matrixFile,
 		samples: samples.join(','),
-		filter_extreme_values: true,
-		num_genes: q.maxGenes,
-		param: 'var'
+		filter_extreme_values: filter_extreme_values,
+		num_genes: q.maxGenes
 	}
+
+	if (q?.filter_type?.type) input_json['param'] = q.filter_type.type
```

Comment:
q is always present
replace with input_json.param=xx

Reply:
I have set default filter_type=iqr. q.filter_type is also an optional variable.